flask/json/tag.py

```diff
@@ -0,0 +1,253 @@
+from base64 import b64decode, b64encode
+from datetime import datetime
+from uuid import UUID
+
+from jinja2 import Markup
+from werkzeug.http import http_date, parse_date
+
+from flask._compat import iteritems, text_type
+from flask.json import dumps, loads
+
+
+class JSONTag(object):
+    """Base class for defining type tags for :class:`TaggedJSONSerializer`."""
+
+    __slots__ = ('serializer',)
+    key = None
+    """The tag to mark the serialized object with. If ``None``, this tag is
```

Comment:
AFAIK the most common sphinx way of documenting a class-level variable is this:

```python
#: lorem ipsum
#: blah blah blah
foo = None
```

At least I find this "dangling" docstring quite ugly and I think some editors even highlight it as a no-op statement  and/or a docstring that's not where it should be/